src/main/java/com/cinema/model/MovieSession.java

```diff
@@ -22,6 +23,7 @@
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "cinema_halls_id")
```

Comment:
let's better use singular here in `@JoinColumn` annotation

Reply:
done